hw_5.py

```diff
@@ -0,0 +1,98 @@
+# 1. Из текстового файла удалить все слова, содержащие от трех до пяти символов,
+# но при этом из каждой строки должно быть удалено только четное количество таких слов.
+
+def word_in_file_deleter(f_name):
+    with open(f_name, "r") as f:
+        lines = [line.split() for line in f]
+        new_lines = []
+    for line in lines:
+        new_line = list(filter(lambda w: 2 < len(w) < 6, line))
+        n_true = len(new_line)
+        if n_true > 1:
+            if n_true % 2 != 0: new_line.pop(-1)
+            for word in new_line:
+                line.remove(word)
+        new_lines.append(line)
+
+    # If we need to rewrite new lines in a file:
+    with open(f_name, "w") as f:
+        for line in new_lines:
+            print(' '.join(line), file=f)
```

Comment:
use `write() `method, `print` is for printing something in output but with the possibility to redirect output to file.